examples/lightgbm/pixi.toml

```
@@ -0,0 +1,14 @@
+[project]
+name = "lightgbm"
+version = "0.1.0"
+description = "LightGBM demo for machine learning project"
+authors = ["fkt <deepthought@postech.ac.kr>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
```

```suggestion
platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
```
We always try to make the examples as multi platform as possible. We can test it on all systems. Adding this does require to rerender the `pixi.lock` Could you do that?